src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -98,7 +98,7 @@ private JValue TransformScalar(JsonSchema schema, Document file, Context context
                         file,
                         dependencyResolver,
                         buildChild,
-                        null,
+                        context.MonikerProvider,
```

Comment:
>context.MonikerProvider [](start = 24, length = 23)

we haven't design the version for sdp, keep this unchanged?

Reply:
Versioning should be the same for both sdp and conceptual, or the markdown pipeline should be the same for sdp and conceptual.